pkg/apis/allocation/v1/gameserverallocation.go

```diff
@@ -514,6 +568,13 @@ func (gsa *GameServerAllocation) ApplyDefaults() {
 			gsa.Spec.Selectors[i].ApplyDefaults()
 		}
 	}
+
+	if runtime.FeatureEnabled(runtime.FeatureCountsAndLists) {
+		// TODO: Do we need to make an empty slice of Priorities here if gsa.Spec.Priorities == nil?
```

Comment:
@markmandel Any idea if I need to create an empty slice of Priorities here if gsa.Spec.Priorities == nil? That seems to be a common pattern, but not sure if it's necessary here?

Reply:
If you `range` over a nil array I'm pretty sure it's fine (and `append` works fine too) - so probably not.

Time will tell however.

Reply:
OK, I'll delete these lines in that case.